bin/ansible-lint

```diff
@@ -48,6 +48,9 @@ def main(args):
     parser.add_option('-p', dest='parseable',
                       default=False, action='store_true',
                       help="parseable output in the format of pep8")
+    parser.add_option('-c', dest='colored',
+                      default=False, action='store_true',
```

Comment:
I'd favour making the default whether the terminal can handle colour codes perhaps using something like 

```
default=hasattr(sys.stdout, 'isatty') and sys.stdout.isatty()
```

Solution from: http://stackoverflow.com/a/22254892

And then change this to `--nocolor/--nocolour` to switch colour off (I'll assume that the above detection is good enough that no one will want to switch colour on if the autodetection doesn't work)